apps/web/src/services/iam/composables/use-user-group-list-query.ts

```diff
@@ -26,7 +26,7 @@ export const useUserGroupListQuery = ({ params }: { params: ComputedRef<UserGrou
 
     return {
         userGroupAPI,
-        userGroupListData: computed(() => userGroupListData?.value?.results || []),
+        userGroupListData: computed(() => userGroupListData?.value || []),
```

Comment:
nitpick: Since you've set the `results` value to `[]` in `initialData`, this part doesn't seem necessary.